### PR TITLE
meson-python 0.13.2 (new formula) scipy 1.11.2

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
scipy test still failed, and meson-python formula need some improvements. :)

File: Formula/s/scipy.rb [hunk 1]
```diff
@@ -51,13 +55,13 @@ def install
     Pathname("site.cfg").write config
 
     site_packages = Language::Python.site_packages(python3)
+    ENV.prepend_path "PATH", Formula["libcython"].opt_libexec/"bin"
```
Comment (chenrui333) [inline Formula/s/scipy.rb:58]: 👍

File: Formula/m/meson-python.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  url "https://files.pythonhosted.org/packages/ed/77/786ac8dcc8bc39a927527ba68a016bf9bd8f7ffe5c3622597ad16cd218af/meson_python-0.13.2.tar.gz"
+  sha256 "80bc9de898acd36eb4b945afaaf7a2b4ca00189c51870d535e329761910cf8ea"
+  license "MIT"
+
+  depends_on "python-flit-core" => :build
+  depends_on "python@3.10" => [:build, :test]
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.9" => [:build, :test]
+  depends_on "meson"
+  depends_on "python-packaging"
```
Comment (chenrui333) [inline Formula/m/meson-python.rb:13]: why do we need this?

Comment (branchvincent) [reply Formula/m/meson-python.rb:13]: its a dependency of `pyproject-metadata`:

```console
$ pipgrip --tree meson-python
meson-python (0.13.2)
├── meson>=0.63.3 (1.2.1)
└── pyproject-metadata>=0.7.1 (0.7.1)
    └── packaging>=19.0 (23.1)
```

Comment (chenrui333) [reply Formula/m/meson-python.rb:13]: got you, thanks!

**Review by p-linnane — APPROVED** [review]

**Pull request merged**